### DISCUSSION TITLE
Issues in migrating from Specflow to Reqnroll

### DISCUSSION THREAD

**Discussion by RajeshPoola** [reqnroll/Reqnroll#372] [SpecFlow migration Q&A]
In our project, we used the 'Specflow.Assist.Dynamic' library to facilitate the dynamic creation of sets and instances. As we are planning to migrate our projects to Reqnroll, which lacks compatibility with dynamic table handling, I forked the 'Specflow.Assist.Dynamic' repository and attempted to adapt it for use with Reqnroll tables. However, I am encountering issues because it does not recognize the NUnit.Framework. I'm not sure what I'm missing.

![image](https://github.com/user-attachments/assets/6c899119-0cfa-4172-8a88-3fa7a8d04e4b)

Project file
`    <ItemGroup>
    <PackageReference Include="Dynamitey" Version="3.0.3" />
    <PackageReference Include="FluentAssertions" Version="7.0.0" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.12.0" />
    <PackageReference Include="Reqnroll" Version="2.2.1" />
    <PackageReference Include="Reqnroll.NUnit" Version="2.2.1" />
    <PackageReference Include="NUnit" Version="4.3.1" />
    <PackageReference Include="NUnit3TestAdapter" Version="4.6.0" />
  </ItemGroup>`

**Comment by olegKoshmeliuk** [comment]
looks like you have clash of namespaces
Reqnroll.NUnit (goes from NUnit generator) with NUnit Framework. 
In test project remove `Reqnroll` namespace part. it will fix build error
btw: I have successfully ported the package you can try 
https://github.com/olegKoshmeliuk/Reqnroll.Assist.Dynamic source code 
https://www.nuget.org/packages/ReqnrollExtra.Assist.Dynamic nuget package
